### DISCUSSION TITLE
Cannot brows to audio files

### DISCUSSION THREAD

**Discussion by AlexKLMan** [yermak/AudioBookConverter#564] [General]
I'm new to the program, so I am so sorry if this is something I am doing wrong, I tried adding audio files to the program, I tried both the folder and the files but I cannot locate my books in mp3 at all. the folder where I put my books is just empty, I'm not sure what I'm doing wrong here. I even created a new folder and moved my books in there and selected the newly created folder, but nothing happens.
I'm running on Windows 10 version 6.2.10.
Thank you.

**Comment by yermak** [comment]
I don't understand if this is a bug or you need a guidance how to use program.
Moved it from issues to discussion
